app/scripts/modules/core/delivery/create/CreatePipelineModal.tsx

```diff
@@ -200,22 +200,26 @@ export class CreatePipelineModal extends React.Component<ICreatePipelineModalPro
                   <b>Type</b>
                 </div>
                 <div className="col-md-7">
-                  <Select options={[{label: 'Pipeline', value: false}, {label: 'Strategy', value: true}]}
-                          clearable={false}
-                          value={this.state.command.strategy ? {label: 'Strategy'} : {label: 'Pipeline'}}
-                          onChange={this.handleTypeChange}/>
+                  <Select
+                    options={[{label: 'Pipeline', value: false}, {label: 'Strategy', value: true}]}
+                    clearable={false}
+                    value={this.state.command.strategy ? {label: 'Strategy'} : {label: 'Pipeline'}}
+                    onChange={this.handleTypeChange}
```

Comment:
Driven by lint rules. It comes as a part of a wholesale `jsx-alignment` rule. I like every other bit of the rule (if multi-line, can't have more than one attribute per line, closing `/>` on new line aligned with opening `<`...), and I'm indifferent to the attribute on the next line. I assume the reason is for simplifying diffing.